src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,61 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    imgUrl,
+    imdbUrl,
+    title,
+    description,
+  } = movie;
+
+  return (
+    <ul className="movies">
```

Comment:
now you are creating a new `ul` for each movie card.

so, just move `ul` tag to `MovieList` component and leave `li` tag here